### ISSUE TITLE
README.md not included in dist?

### ISSUE THREAD

**Issue by jefftriplett** [rsinger86/django-lifecycle#21]
I ran into this earlier and it looks like maybe your README.md is not being included in 0.4.1: 

```$ pip install -U django-lifecycle
Collecting django-lifecycle
  Using cached https://files.pythonhosted.org/packages/d4/ab/9daddd333fdf41bf24da744818a00ce8caa8e39d93da466b752b291ce412/django-lifecycle-0.4.1.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/pb/j_dpdd4n0858j1ym98g3884r0000gn/T/pip-install-x3_d5nyd/django-lifecycle/setup.py", line 30, in <module>
        long_description=readme(),
      File "/private/var/folders/pb/j_dpdd4n0858j1ym98g3884r0000gn/T/pip-install-x3_d5nyd/django-lifecycle/setup.py", line 7, in readme
        with open("README.md", "r") as infile:
      File "/Users/jefftriplett/.pyenv/versions/3.6.5/lib/python3.6/codecs.py", line 897, in open
        file = builtins.open(filename, mode, buffering)
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/pb/j_dpdd4n0858j1ym98g3884r0000gn/T/pip-install-x3_d5nyd/django-lifecycle/
```

**Comment by notanumber** [comment]
Getting the same error. Took a look inside the package and only seeing:

django-lifecycle-0.4.1/
django-lifecycle-0.4.1/django_lifecycle/
django-lifecycle-0.4.1/django_lifecycle/models.py
django-lifecycle-0.4.1/django_lifecycle/__init__.py
django-lifecycle-0.4.1/setup.cfg
django-lifecycle-0.4.1/setup.py
django-lifecycle-0.4.1/README.txt
django-lifecycle-0.4.1/PKG-INFO
django-lifecycle-0.4.1/django_lifecycle.egg-info/
django-lifecycle-0.4.1/django_lifecycle.egg-info/SOURCES.txt
django-lifecycle-0.4.1/django_lifecycle.egg-info/dependency_links.txt
django-lifecycle-0.4.1/django_lifecycle.egg-info/PKG-INFO
django-lifecycle-0.4.1/django_lifecycle.egg-info/top_level.txt

**Comment by arthexis** [comment]
Same error here.

**Comment by rsinger86** [comment]
Woah, my bad! This should be fixed now in 0.4.2.

**Comment by jefftriplett** [comment]
@rsinger86 0.4.2 works perfectly and thanks again for the urlman fix.
